config/env.development

```diff
@@ -87,6 +87,9 @@ JWT_AUDIENCE=http://localhost
 # How long should a JWT work before it expires
 JWT_EXPIRES_IN=1h
 
+# Example Supabase secrets for development
```

Comment:
I think you can remove this.  Everything in this file isn't suitable for production.